src/service/analytics.service.spec.ts

```diff
@@ -0,0 +1,101 @@
+import { HelpConfig } from '../help-config';
+import { BBHelpAnalyticsService } from './analytics.service';
+
+const demoConfig: HelpConfig = {
+  productId: 'bbHelpTesting'
+};
+
+describe('BBHelpAnalyticsService', () => {
+  let analyticsService: BBHelpAnalyticsService;
+  let originalTimeout: number;
+  let mixpanelSpy: any;
+
+  beforeEach(() => {
+    originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000;
+
+    analyticsService = new BBHelpAnalyticsService();
+
+    mixpanelSpy = spyOn<any>(analyticsService, 'getMixpanel').and.returnValue({
+      bb_help_widget: {
+        identify: () => { return; },
+        register: (object: any) => { return; },
+        track: (eventName: any, payload: any) => { return; }
+      },
+      init: (key: any, config: any, name: any) => { return; }
+    });
+  });
+
+  afterEach(() => {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = originalTimeout;
+  });
+
+  it('should set up the mixpanel', (done) => {
+    spyOn<any>(analyticsService, 'registerSuperProperties').and.callThrough();
+
+    analyticsService.setupMixpanel(demoConfig.productId);
+
+    expect(analyticsService['superProperties']).toEqual({
+      'Referring Service Name': 'bbHelpTesting'
+    });
+    expect(analyticsService['analyticsClient']).toBeDefined();
+    expect(analyticsService['registerSuperProperties']).toHaveBeenCalled();
+    done();
+  });
+
+  it('should set up the mixpanel with unsupported error code', (done) => {
```

Comment:
I am guessing this test is left over from some other changes made.  Remove this one and I think we are good to go